### PR TITLE
Fix/BFL-53

### LINKED ISSUE #53: Rewrite building managing
Rewrite current mechanic for buildings creating and managing for future saving and loading

### PR DESCRIPTION
**Pull request by weterbogov**
Resolved #53

### CODE REVIEW

File: Assets/Project Scope/Scripts/BuildingLabel.cs [hunk 1]
```diff
@@ -0,0 +1,7 @@
+using UnityEngine;
+
+public class BuildingLabel: MonoBehaviour
```
Comment (Verpad) [inline Assets/Project Scope/Scripts/BuildingLabel.cs]: Why do we need whole class for storing building label?

Comment (weterbogov) [reply Assets/Project Scope/Scripts/BuildingLabel.cs]: Can you suggest another solution for global constants?

Comment (Verpad) [reply Assets/Project Scope/Scripts/BuildingLabel.cs]: @weterbogov will you store here more parameters in the future?

Comment (weterbogov) [reply Assets/Project Scope/Scripts/BuildingLabel.cs]: @Verpad it is possible. As I think, we should have special container for all buildings and store them by key. This key we can get from BuildingLabel class.

Comment (weterbogov) [reply Assets/Project Scope/Scripts/BuildingLabel.cs]: @Verpad it is possible, but for now this is enough

**Pull request merged**